mythplugins/configure

```diff
@@ -852,6 +868,16 @@ if enabled music ; then
       echo "#define HAVE_CDIO 1" >> ./mythmusic/mythmusic/config.h
       echo "CONFIG += cdio" >> ./mythmusic/mythmusic/config.pro
       echo "        libcdio        support will be included in MythMusic"
+      if enabled musicbrainz ; then
+        echo "#define HAVE_MUSICBRAINZ 1" >> ./mythmusic/mythmusic/config.h
+        echo "CONFIG += link_pkgconfig" >> ./mythmusic/mythmusic/config.pro
+        echo "CONFIG += musicbrainz" >> ./mythmusic/mythmusic/config.pro
+        echo "PKGCONFIG += libdiscid libmusicbrainz5 libcoverart" >> ./mythmusic/mythmusic/config.pro
```

Comment:
Should the linked libraries not be libmusicbrainz5cc and libcoverartcc for c++?

Reply:
These are the pkgconfig names used to get the respective library names and include paths from pkgconfig. As example taken from the musicbrainz library documentation (http://metabrainz.github.io/libmusicbrainz/5.1.0/):
`g++ -o test_app test_app.cpp pkg-config libmusicbrainz5 --cflags --libs`

Reply:
Strange if I try with what you have everything compiles and links but when I try to run it I get missing symbol errors for both libmusicbrainz5 and libcoverart.

Change to  libmusicbrainz5cc and libcoverartcc and everything works fine?
 
According to the description of the Ubuntu packages libmusicbrainz5cc2v5 is the shared library and libmusicbrainz5-2 is the shared library for the C wrapper functions.

pkg-config contains entries for both libmusicbrainz5 and libmusicbrainz5cc and also for libcoverart and libcoverartcc.

Reply:
Oh well :-/, I am on ArchLinux and there is only a single entry for both libmusicbrainz and libcoverart.  Any suggestion on how to solve that?

Reply:
After investigating a bit, it seems the split of the c and c++ interfaces was done after the last official release. ArchLinux is using the source tarball of the last official release, so there is only one library present.

Reply:
Ok I added a switch to support both version combined and split libs. Could you please try if that works on your side?

Reply:
Thanks it now works for me also.